wbooks/src/app/components/BookDetail/index.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { Component } from 'react';
+import { Text, View, Image } from 'react-native';
+import styles from './styles'
+
+export default function BookDetail({title, subtitle, url}: {title: string, subtitle: string, url: string}) {
+  return (
+    <View style={styles.container}>
+      <Image source={{
+          uri: url,
+        }} style={styles.image} />
```

Comment:
Add `resizeMode` for the [Image](https://reactnative.dev/docs/image.html#resizemode). For example:
`<Image source={{ uri: url }} style={styles.image} resizeMode="contain" />`